checker/tests/nullness-extra/multiple-errors/Makefile

```diff
@@ -1,6 +1,8 @@
-.PHONY: all
+.PHONY: all clean
 
 all:
-	rm -f Out.txt
 	-$(JAVAC) -processor org.checkerframework.checker.nullness.NullnessChecker *.java > Out.txt 2>&1
 	diff -u Expected.txt Out.txt
+
+clean:
+	rm -f Out.txt
```

Comment:
No `.class` here?

Reply:
I think this test don't generate class file because the checker will issue error